### PR TITLE
Remove duplicate `checkPutObjectArgs` in PutObject

### LINKED ISSUE #7384: Duplicated bucket existence checking operation
Duplicated bucket existence checking operation
## Expected Behavior

Check once is enough
## Current Behavior

Check twice the same thing exactly 
## Possible Solution

Remove one
## Steps to Reproduce (for bugs)

one of the `checkPutObjectArgs` is in here:
```
func (xl xlObjects) PutObject(ctx context.Context, bucket string, object string, data *PutObjReader, opts ObjectOptions) (objInfo ObjectInfo, err error)
```
the other is in 
```
func (xl xlObjects) putObject(ctx context.Context, bucket string, object string, r *PutObjReader, opts ObjectOptions) (objInfo ObjectInfo, err error) {
```

## Your Environment

* Version used (`minio version`): RELEASE.2019-02-26T19-51-46Z
* Environment name and version (e.g. nginx 1.9.1):
* Server type and version:
* Operating System and version (`uname -a`):
* Link to your project:

### PR DESCRIPTION
**Pull request by Praveenrajmani**
## Description
We were double checking the input args in PutObject.

## Motivation and Context
Fixes  #7384

## Regression
No

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#b93928d3-2881-4a4a-b272-044311549624) |
| mint-compression-dist-xl.sh | :x: [more...](#809c0d79-213d-4083-a4ed-1b9ed8d62802) |
| mint-compression-fs.sh | :x: [more...](#6c7e071f-f9a7-4449-9c7f-3f9b9aa6ef56) |
| mint-dist-xl.sh | :x: [more...](#b8018810-86fa-4295-a97a-141376040b9b) |
<a name="809c0d79-213d-4083-a4ed-1b9ed8d62802"></a>
### 7396-ce794ed/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:32460
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 1db252ecba54:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 4 seconds
(5/14) Running awscli tests ... done in 1 minutes and 48 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 32 seconds
(8/14) Running minio-dotnet tests ... done in 34 seconds
(9/14) Running minio-go tests ... done in 59 seconds
(10/14) Running minio-java tests ... FAILED in 27 seconds
{
  "name": "minio-java",
  "function": "putObject(String bucketName, String objectName, InputStream body, long size, String contentType)",
  "args": "contentType: application/javascript",
  "duration": 21119,
  "status": "FAIL",
  "error": "java.net.SocketException: Connection reset >>> [java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:115), java.net.SocketOutputStream.write(SocketOutputStream.java:155), okio.Okio$1.write(Okio.java:78), okio.AsyncTimeout$1.write(AsyncTimeout.java:179), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:171), okio.RealBufferedSink.write(RealBufferedSink.java:41), okhttp3.internal.http1.Http1Codec$FixedLengthSink.write(Http1Codec.java:286), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:171), okio.RealBufferedSink.write(RealBufferedSink.java:109), io.minio.HttpRequestBody.writeTo(HttpRequestBody.java:74), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:62), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:120), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:185), okhttp3.RealCall.execute(RealCall.java:69), io.minio.MinioClient.executeReq(MinioClient.java:1097), io.minio.MinioClient.execute(MinioClient.java:1056), io.minio.MinioClient.executePut(MinioClient.java:1399), io.minio.MinioClient.executePut(MinioClient.java:1421), io.minio.MinioClient.putObject(MinioClient.java:3994), io.minio.MinioClient.putObject(MinioClient.java:4086), io.minio.MinioClient.putObject(MinioClient.java:3960), FunctionalTest.testPutObjectUnknownStreamSize(FunctionalTest.java:556), FunctionalTest.putObject_test8(FunctionalTest.java:582), FunctionalTest.runTests(FunctionalTest.java:2998), FunctionalTest.main(FunctionalTest.java:3154)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="b8018810-86fa-4295-a97a-141376040b9b"></a>
### 7396-ce794ed/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:31487
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d4665c9ff5cf:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 1 minutes and 50 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... done in 29 seconds
(8/14) Running minio-dotnet tests ... done in 42 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 4 seconds
(10/14) Running minio-java tests ... FAILED in 15 seconds
{
  "name": "minio-java",
  "function": "putObject(String bucketName, String objectName, InputStream body, long size, String contentType)",
  "args": "contentType: application/javascript",
  "duration": 7589,
  "status": "FAIL",
  "error": "java.net.SocketException: Connection reset >>> [java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:115), java.net.SocketOutputStream.write(SocketOutputStream.java:155), okio.Okio$1.write(Okio.java:78), okio.AsyncTimeout$1.write(AsyncTimeout.java:179), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:171), okio.RealBufferedSink.write(RealBufferedSink.java:41), okhttp3.internal.http1.Http1Codec$FixedLengthSink.write(Http1Codec.java:286), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:171), okio.RealBufferedSink.write(RealBufferedSink.java:109), io.minio.HttpRequestBody.writeTo(HttpRequestBody.java:74), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:62), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:120), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:185), okhttp3.RealCall.execute(RealCall.java:69), io.minio.MinioClient.executeReq(MinioClient.java:1097), io.minio.MinioClient.execute(MinioClient.java:1056), io.minio.MinioClient.executePut(MinioClient.java:1399), io.minio.MinioClient.executePut(MinioClient.java:1421), io.minio.MinioClient.putObject(MinioClient.java:3994), io.minio.MinioClient.putObject(MinioClient.java:4086), io.minio.MinioClient.putObject(MinioClient.java:3960), FunctionalTest.testPutObjectUnknownStreamSize(FunctionalTest.java:556), FunctionalTest.putObject_test8(FunctionalTest.java:582), FunctionalTest.runTests(FunctionalTest.java:2998), FunctionalTest.main(FunctionalTest.java:3154)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="6c7e071f-f9a7-4449-9c7f-3f9b9aa6ef56"></a>
### 7396-ce794ed/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:31782
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp c30a5258c4f5:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-k3s6wzeglw04qu",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 313,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.61:31782/aws-sdk-go-test-k3s6wzeglw04qu: dial tcp 72.28.97.61:31782: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="b93928d3-2881-4a4a-b272-044311549624"></a>
### 7396-ce794ed/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:30702
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 28a9c96bdcd7:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 1 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-ccg4lbd2pcxce9",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 350,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:30702/aws-sdk-go-test-ccg4lbd2pcxce9: dial tcp 72.28.97.57:30702: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Review by poornas — COMMENTED** [review]
CopyObject also uses putObject() @Praveenrajmani, so you might want to checkPutObjectArgs for destination side in CopyObject

**Comment by Praveenrajmani** [comment]
> CopyObject also uses putObject() @Praveenrajmani, so you might want to checkPutObjectArgs for destination side in CopyObject

Oh yes. Thanks @poornas . 

And i see that we do lockless `putObject` call instead of locked `PutObject` call in CopyObject. Will fix that in the same PR.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#18bcaf99-37ee-45a6-a84e-8e1ed84ed1f2) |
| mint-compression-dist-xl.sh | :x: [more...](#d06d65c9-8f68-4e22-953e-3ece72cfdc24) |
| mint-fs.sh | :x: [more...](#43fdf85c-acde-460b-a01e-4d3cebc69e11) |
| mint-dist-xl.sh | :x: [more...](#1a381859-cf12-400d-a7f6-e12bd0e0ebeb) |
<a name="18bcaf99-37ee-45a6-a84e-8e1ed84ed1f2"></a>
### 7396-c0db824/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:32005
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 20f57381e045:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 1 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-atdy46golnrmna",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 273,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:32005/aws-sdk-go-test-atdy46golnrmna: dial tcp 72.28.97.57:32005: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="d06d65c9-8f68-4e22-953e-3ece72cfdc24"></a>
### 7396-c0db824/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:30357
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 30d1e0b4f2ee:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-19ulcoe1slkqmf",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 334,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.53:30357/aws-sdk-go-test-19ulcoe1slkqmf: dial tcp 72.28.97.53:30357: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="43fdf85c-acde-460b-a01e-4d3cebc69e11"></a>
### 7396-c0db824/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:30440
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d1e62be68c4a:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 2 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-qzok12dco31id2",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 120340,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.60:30440/aws-sdk-go-test-qzok12dco31id2: dial tcp 72.28.97.60:30440: i/o timeout",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="1a381859-cf12-400d-a7f6-e12bd0e0ebeb"></a>
### 7396-c0db824/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:31303
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7611dba481c4:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 1 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-f6gy9mrnarsabh",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 267,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.53:31303/aws-sdk-go-test-f6gy9mrnarsabh: dial tcp 72.28.97.53:31303: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by Praveenrajmani** [comment]
PTAL @poornas @balamurugana

File: cmd/fs-v1.go [hunk 1]
```diff
@@ -853,10 +854,6 @@ func (fs *FSObjects) putObject(ctx context.Context, bucket string, object string
 		return fsMeta.ToObjectInfo(bucket, object, fi), nil
 	}
 
-	if err = checkPutObjectArgs(ctx, bucket, object, fs, data.Size()); err != nil {
```
Comment (poornas) [inline cmd/fs-v1.go:857]: object args need to be checked  for destination object in fs CopyObject since you have commented this out.

Comment (Praveenrajmani) [reply cmd/fs-v1.go:857]: Added :+1:  PTAL

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :x: [more...](#34ba7cc6-91ef-413e-8424-ee7bfa47d166) |
<a name="34ba7cc6-91ef-413e-8424-ee7bfa47d166"></a>
### 7396-407f218/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:31948
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 6a0572c68422:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-xpzktjsz93xxlc",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 290,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.56:31948/aws-sdk-go-test-xpzktjsz93xxlc: dial tcp 72.28.97.56:31948: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Review by poornas — DISMISSED** [review]
LGTM

**Comment by harshavardhana** [comment]
There is a bug in this PR which will cause deadlock

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#ce564855-f1e5-4b42-9b12-7a7ebd0f211e) |
| mint-large-bucket.sh | :x: [more...](#4e027d3c-65e5-4c8e-8141-185b79e28280) |
| mint-compression-dist-xl.sh | :x: [more...](#b3bcc77e-6150-47be-b96a-c9b6ba8d2452) |
| mint-worm.sh | :x: [more...](#ce4e42fe-6f6a-4113-b154-a80dbed1917c) |
| mint-dist-xl.sh | :x: [more...](#dc9dae01-7c38-4ce9-9f85-10b16c33d6bc) |
<a name="dc9dae01-7c38-4ce9-9f85-10b16c33d6bc"></a>
### 7396-cb3115e/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:32675
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e2d328dcd60f:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-hzzprihenuwdz4",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 333,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.53:32675/aws-sdk-go-test-hzzprihenuwdz4: dial tcp 72.28.97.53:32675: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="ce4e42fe-6f6a-4113-b154-a80dbed1917c"></a>
### 7396-cb3115e/mint-worm.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31313
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            worm
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp f29c72ae021a:/mint/log /tmp/mint-logs'

(1/1) Running worm tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "bucket-1mhjjblzlu0x2yp22j3uhksz02vbqyu1dnm7kozyixuey9sedxk24gy",
    "expiry": 60000000000,
    "objectName": "testObject"
  },
  "duration": 3059,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:31313/bucket-1mhjjblzlu0x2yp22j3uhksz02vbqyu1dnm7kozyixuey9sedxk24gy: dial tcp 72.28.97.57:31313: connect: no route to host",
  "function": "PutAndDelete",
  "message": "WORM_MODE ON  - CreateBucket Failed",
  "name": "test worm mode",
  "status": "FAIL"
}

Executed 0 out of 1 tests successfully.
```
<a name="b3bcc77e-6150-47be-b96a-c9b6ba8d2452"></a>
### 7396-cb3115e/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:30581
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp f5d57bd08b47:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-a990bpblbzxzxo",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 324,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.61:30581/aws-sdk-go-test-a990bpblbzxzxo: dial tcp 72.28.97.61:30581: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="ce564855-f1e5-4b42-9b12-7a7ebd0f211e"></a>
### 7396-cb3115e/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:30996
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 32625b402887:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-dj4e6scji1whfx",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 301,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.56:30996/aws-sdk-go-test-dj4e6scji1whfx: dial tcp 72.28.97.56:30996: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="4e027d3c-65e5-4c8e-8141-185b79e28280"></a>
### 7396-cb3115e/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:30351
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 422f0c46f6e0:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-u3wgrwx0sf6w6p",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 295,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.51:30351/aws-sdk-go-test-u3wgrwx0sf6w6p: dial tcp 72.28.97.51:30351: connect: connection refused",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by harshavardhana** [comment]
@abperiasamy suggested that this should be taken in as part of performance improvement.. 

@deekoder moving this as high priority as well

@Praveenrajmani please rebase this PR with master

@balamurugana can you review this PR?

**Comment by Praveenrajmani** [comment]
Resolved the conflicts. PTAL @poornas @balamurugana

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#6f2fae9b-c5ac-481b-972b-89a6646be271) |
<a name="6f2fae9b-c5ac-481b-972b-89a6646be271"></a>
### 7396-7b729b3/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:30047
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 81b7041135e6:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-ues4agt9hleutm",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360018,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 159E2E4BB4CEEAF7, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Review by balamurugana — APPROVED** [review]

**Review by poornas — APPROVED** [review]
LGTM

**Pull request merged**